ios/Classes/utils/ConfigurationMapper.swift

```diff
@@ -2,17 +2,16 @@
 import Adyen
 import PassKit
 
-class ConfigurationMapper {
-    func createDropInConfiguration(dropInConfigurationDTO: DropInConfigurationDTO) throws -> DropInComponent.Configuration {
+extension DropInConfigurationDTO {
+    func createDropInConfiguration(payment: Payment?) throws -> DropInComponent.Configuration {
         let dropInConfiguration = DropInComponent.Configuration(
-            allowsSkippingPaymentList: dropInConfigurationDTO.skipListWhenSinglePaymentMethod,
-            allowPreselectedPaymentView: dropInConfigurationDTO.showPreselectedStoredPaymentMethod
+            allowsSkippingPaymentList: skipListWhenSinglePaymentMethod,
+            allowPreselectedPaymentView: showPreselectedStoredPaymentMethod
         )
-        
-        dropInConfiguration.paymentMethodsList.allowDisablingStoredPaymentMethods =
-            dropInConfigurationDTO.isRemoveStoredPaymentMethodEnabled
-        
-        if let cardConfigurationDTO = dropInConfigurationDTO.cardConfigurationDTO {
+
+        dropInConfiguration.paymentMethodsList.allowDisablingStoredPaymentMethods = isRemoveStoredPaymentMethodEnabled
+
+        if let cardConfigurationDTO {
```

Comment:
what about moving the content of this if to a different function? `func cardConfiguration(from dto:) -> CardComponent.Configuration`